p2p/agent.go

```diff
@@ -236,59 +269,96 @@ func (p *agent) Start(ctx context.Context) error {
 		return errors.Wrap(err, "error when instantiating Agent host")
 	}
 
-	if err := host.AddBroadcastPubSub(ctx, _broadcastTopic+p.topicSuffix, func(ctx context.Context, data []byte) (err error) {
-		// Blocking handling the broadcast message until the agent is started
-		<-ready
-		var (
-			peerID    string
-			broadcast iotexrpc.BroadcastMsg
-			latency   int64
-		)
-		skip := false
-		defer func() {
-			// Skip accounting if the broadcast message is not handled
-			if skip {
-				return
+	if err := host.AddBroadcastPubSub(
+		ctx,
+		_broadcastTopic+p.topicSuffix,
```

Comment:
consider add a `DefaultTopic()` for `host`, which returns `_broadcastTopic+p.topicSuffix`, so it is like
```
host.AddBroadcastPubSub(ctx, host.DefaultTopic())
```
this will also make it easier for later PR to split topics for consensus/action/block